tests/sub-menu-wrapper-test.js

```diff
@@ -1,29 +0,0 @@
-
-
-import React from 'react';
```

Comment:
Unlike `sub-menu-tests.js`,  I don't see the new file of this `SubMenuWrapper` component. Is that intentional or is it because of a slipped `git add`? :)

Reply:
as we discussed, this test file wasn't run before, and is about a component that doesn't seem present anymore.